07-routing-gossip.md

```diff
@@ -178,7 +198,10 @@ valid signature using `node-id` of the double-SHA256 of the entire
 message following the `signature` field (including unknown fields
 following `alias`), and MUST NOT further process the message.
 
-The receiving node SHOULD ignore `ipv6` if `port` is zero.
+The receiving node SHOULD ignore the first `address descriptor` which
```

Comment:
Yes, it's implied in the "[addrlen:addresses]" description, but I've made it explicit.